go/arrow/ipc/file_reader.go

```diff
@@ -312,16 +312,23 @@ func (f *FileReader) ReadAt(i int64) (array.Record, error) {
 
 func newRecord(schema *arrow.Schema, meta *memory.Buffer, body ReadAtSeeker) array.Record {
 	var (
-		msg = flatbuf.GetRootAsMessage(meta.Bytes(), 0)
-		md  flatbuf.RecordBatch
+		msg   = flatbuf.GetRootAsMessage(meta.Bytes(), 0)
```

Comment:
what applied this formatting change? Is go formatting not already checked in CI?

Reply:
I don't know if go formatting is checked in the CI already, but the formatting change here happened because my IDE is set to automatically apply gofmt formatting whenever I save. So this formatting change happened because of applying proper formatting.